src/modules/PositionModule.cpp

```diff
@@ -230,4 +230,42 @@ int32_t PositionModule::runOnce()
     }
 
     return 5000; // to save power only wake for our callback occasionally
+}
+
+void PositionModule::handleNewPosition()
+{
+    meshtastic_NodeInfoLite *node = nodeDB.getMeshNode(nodeDB.getNodeNum());
+    const meshtastic_NodeInfoLite *node2 = service.refreshLocalMeshNode(); // should guarantee there is now a position
+    // We limit our GPS broadcasts to a max rate
+    uint32_t now = millis();
+    uint32_t intervalMs = getConfiguredOrDefaultMs(config.position.position_broadcast_secs, default_broadcast_interval_secs);
```

Comment:
src/modules/PositionModule.cpp: In member function 'void PositionModule::handleNewPosition()':
src/modules/PositionModule.cpp:241:14: warning: unused variable 'intervalMs' [-Wunused-variable]
     uint32_t intervalMs = getConfiguredOrDefaultMs(config.position.position_broadcast_secs, default_broadcast_interval_secs);